web/js/ui/mouse.js

```diff
@@ -0,0 +1,112 @@
+import * as d3 from 'd3';
```

Comment:
This seems to be the usual way to import `d3`, but it sounds like [there might be some problems with importing `d3.event`](https://github.com/d3/d3/issues/2733) which is used in this file. I think it's also possible to import just the functions we need from `d3`, but we should probably wait until we get our whole codebase modularized before tinkering too much with this.